core-plugins/src/test/java/co/cask/hydrator/plugin/transform/ProjectionTransformTest.java

```diff
@@ -21,7 +21,8 @@
 import co.cask.cdap.api.data.schema.Schema;
 import co.cask.cdap.etl.api.Transform;
 import co.cask.cdap.etl.api.TransformContext;
-import co.cask.hydrator.plugin.common.MockEmitter;
+import co.cask.hydrator.common.mock.MockEmitter;
```

Comment:
since we are already in "mock" package, might be better to name the classes just Emitter, etc. what do you think?